src/xrSound/stdafx.cpp

```diff
@@ -15,4 +15,6 @@
 #pragma comment(lib, "OpenAL32B.lib")
 #pragma comment(lib, "dsoundb.lib")
 //#pragma comment(lib, "xrAPI.lib")
+#elif DEBUG
+#pragma comment(lib, "xrCDB.lib") // Xottab_DUTY: For some reason it wont compile without it.
```

Comment:
I don't agree with that reasoning, pulling in extra library inputs for the linker is not a problem at all. You don't even need to limit it to the Debug configuration. Only objects from the library that are actually used will be linked to.

Please move this to the project file, then we can merge this PR.